arch/x86/chunkset_avx2.c

```diff
@@ -32,7 +32,7 @@ static inline void chunkmemset_8(uint8_t *from, chunk_t *chunk) {
 }
 
 static inline void chunkmemset_16(uint8_t *from, chunk_t *chunk) {
-    *chunk = _mm256_broadcastsi128_si256(_mm_loadu_si128((__m128i*)from));
+    *chunk = _mm256_broadcastsi128_si256(*((__m128i*)from));
```

Comment:
An inline comment might be a good idea so nobody tries it again.

Reply:
This change actually has the effect of doing exactly what we didn't want - forcing an aligned load at an unaligned offset. I'll need to redo this PR completely.

Short of inline assembly to do what the intrinsic should have been defined as all along the best we could do is use inserti128 if we detect the broken compiler.